### PR TITLE
Move all terraform files and folders out of root dir into terraform dir

### PR DESCRIPTION
**Pull request by anthonyhashemi**
https://trello.com/c/RqyUEeFz/380-reorganise-enrichment-repo-to-help-navigating-it

Changes in this PR:
- Move all terraform files and folders out of root dir into terraform dir
- Updated all terraform commands appropriately
- Updated reference to lambdas source path

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -150,14 +150,14 @@ As we use AWS Aurora, there is no multi-AZ functionality. Instead, “Aurora aut
...
 We also run a manual snapshot of the cluster at midday (UTC) each day. This is cron-based from Amazon Eventbridge that triggers a [lambda](/lambda/db_backup/index.py). DB backups are shown in the RDS console under manual snapshots.
 
-## 8 Infrastructure
+## 8 Terraform Infrastructure
 
 Here are some brief notes on extending the infrastructure.
 
-- The file `main.tf` at the root of the repo will invoke each of the modules, more of the same services can be created by adding to those modules. If more modules are created then `main.tf` will need to be extended to invoke them.
-- Adding an S3 bucket is done by invoking the `secure_bucket` module, located at `modules/secure_bucket/`, you can see how the existing buckets are created by viewing `modules/lambda_s3/bucket.tf`, new buckets should be created by adding to this file.
+- The file `terraform/main.tf` at will invoke each of the modules, more of the same services can be created by adding to those modules. If more modules are created then `terraform/main.tf` will need to be extended to invoke them.
```
Comment (jacksonj04) [inline README.md:157]:
```suggestion
- The file `terraform/main.tf` will invoke each of the modules, more of the same services can be created by adding to those modules. If more modules are created then `terraform/main.tf` will need to be extended to invoke them.
```

File: terraform/modules/lambda_s3/variables.tf [hunk 2]
```diff
@@ -64,7 +64,7 @@ variable "runtime" {
 
 variable "lambda_source_path" {
   type    = string
-  default = "./lambda/"
+  default = "../lambda/"
```
Comment (jacksonj04) [inline terraform/modules/lambda_s3/variables.tf:67]: Does this need to now be 
```suggestion
  default = "../lambdas/"
```

Comment (anthonyhashemi) [reply terraform/modules/lambda_s3/variables.tf:67]: needs to be `../src/lambdas/`

File: README.md (hunk 1 above)
Comment (anthonyhashemi) [reply README.md:157]: thanks

**Review by jacksonj04 — APPROVED** [review]

**Pull request merged**